src/types.ts

```diff
@@ -0,0 +1,54 @@
+
+export interface UploaderArgs {
+  build?: string                      // Specify the build number manually
+  branch?: string                     // Specify the branch manually
+  dir?: string                        // Directory to search for coverage reports.
+  env?: string                        // Specify environment variables to be included with this build
+  sha?: string                        // Specify the commit SHA mannually
+  file?: string | string[]            // Target file(s) to upload
+  flags: string | string[]            // Flag the upload to group coverage metrics
```

Comment:
```suggestion
  flags?: string | string[]            // Flag the upload to group coverage metrics
```

Disregard this review comment if it was intended for `flags` to be required unlike the other properties.

Reply:
`flags` was not, but after I went though and added it to all the test `input`s, I didn't feel like going back and removing it.

Reply:
👌